release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -2966,6 +2966,10 @@ For more information, see xref:../scalability_and_performance/ztp_far_edge/ztp-d
 +
 If these conditions are not met, the pod can fail at admission with `UnexpectedAdmissionError` error, which is an expected behavior. If the application pod is part of deployments, in case of failure subsequent pods are spun up and ultimately successfully run when devices are suitable to be allocated. (link:https://issues.redhat.com/browse/OCPBUGS-14438[*OCPBUGS-14438*])
 
+* Previously, client TLS (mTLS) was configured on an Ingress Controller, and the certificate authority (CA) in the client CA bundle required more than 1MB of certificate revocation list (CRLs) to be downloaded. The CRL `ConfigMap` object size limitations prevented updates from occurring. As a result of the missing CRLs, connections with valid client certificates may have been rejected with the error `unknown ca`. With this update, the CRL `ConfigMap` for each Ingress Controller no longer exists; instead, each router pod directly downloads CRLs, ensuring connections with valid client certificates are no longer rejected. (link:https://issues.redhat.com/browse/OCPBUGS-13967[*OCPBUGS-13967*])
```

Comment:
I think these might be the same note:

See [Slack](https://redhat-internal.slack.com/archives/CJARLA942/p1687275695169379?thread_ts=1687269752.904019&cid=CJARLA942)

Reply:
While I think these have similar causes, the fixes address two different scenarios and provide two different solutions or changes in behavior.